docs/plugin_advantages.md

```diff
@@ -0,0 +1,52 @@
+# Advantages of using this Plugin
+
+## Handles Windows Init
+One of the core features of the Unity Plugin for EOS, is that it handles
+doing the init code in such a way that the Overlay will work correctly in 
+standalone builds on windows. Due to how the EOS SDK does it's graphics initialization 
+for the overlay, the code that call's into the EOS SDK for init needs to happen _before_
+the Unity finishes doing it's graphics init code. The only place to do this, is in native
+code, in a specially named DLL that gets called before the engine finishes running. 
+If one is using this Plugin, one doesn't have to worry about any of that, as it's handled 
+automatically.
+
+
+## Handles the Unity Editor
+Need to unload and reload the EOS SDK dynamic libraries to ensure new config works
+ensures that any old callbacks can't be called
+
+The Unity Editor has unique behavior that makes integrating the EOS SDK a challenge.
+One such behavior, is that on windows, the Editor never unloads any DLLs. Without additional work,
+this would mean that any changes that need to be done to the EOS configuration would require an
+editor reboot.
+
+To resolve that, we have written additional code to handle loading, and unloading, the EOS SDK manually.
+This both ensures that new configuration changes can be made without restarting the editor, and makes sure
+that any callbacks from previous play sessions aren't called, and that all EOS objects are destroyed between 
+runs in the Editor.
+
+## Provides Samples
+One feature of the Plugin that particularly makes it standout from using just the C# SDK that comes from Epic, 
```

Comment:
Phrasing "just the C# SDK that comes from Epic" might annoy someone.

Reply:
That's a valid point: I do need some better phrasing. I can be less direct with my wording.